### ISSUE TITLE
Specify hlarp as a requirement

### ISSUE THREAD

**Issue by tavinathanson** [hammerlab/discohorts#9]

**Comment by armish** [comment]
@rleonid: can you remind me what was the issue with the biokepi's `hlarp` again? Should we just run within the pipeline instead of dealing with these external uses?

**Comment by tavinathanson** [comment]
@armish since Discohorts supports any pipeline, I don't think we can rely on the pipeline running `hlarp`. I haven't had issues running it locally; I'm curious what the issues are that you're referring to?

**Comment by rleonid** [comment]
[This](https://github.com/hammerlab/biokepi/issues/447) one ?

**Comment by armish** [comment]
@rleonid: oh - I remember that hack! But does the version in the biokepi any different than the one that is currently on the dev channel and installed via opam?

@tavinathanson: was just thinking that it might be much more convenient to just append `|> Bfx.hlarp ...`  to the pipeline scripts instead of pulling hlarp into the discohorts, but never mind my suggestion if it is not that big of hassle at this point ;)

**Comment by tavinathanson** [comment]
@armish good point. I could go either way since `discohorts` is such a collection of hacks right now. Also open to both options working simultaneously.

**Comment by rleonid** [comment]
@armish If we
1. set [extract_alleles](https://github.com/hammerlab/biokepi/blob/master/src/bfx_tools/hlarp.ml#L30) to false in the scripts and
2. update the `biokepi_tracker` [branch](https://github.com/hammerlab/biokepi/blob/374d929281a75de18b96f7896b9fcc9204f0a276/src/environment_setup/biopam.ml#L328) then we can use the master branch. I'd be happy to do the later, but I'm not certain of what else might break. @ihodes  Do you have any idea's?
